browser/src/Searchbox.js

```diff
@@ -33,129 +33,161 @@ const Wrapper = styled.div`
   }
 `
 
+const STRUCTURAL_VARIANT_ID_REGEX = /^(MCNV|INS|DEL|DUP|INV|CPX|OTH)_(\d+|X|Y)_([1-9][0-9]*)$/i
+const isStructuralVariantId = (str) => {
+
+  const match = STRUCTURAL_VARIANT_ID_REGEX.exec(str)
+  if (!match) { return false }
+
+  const chrom = match[2]
+  const chromNumber = Number(chrom)
+  if (!Number.isNaN(chromNumber) && (chromNumber < 1 || chromNumber > 22)) { return false }
+
+  const id = Number(match[3])
+  if (id > 1e9) { return false }
+
+  return true
+}
+
 export const fetchSearchResults = (dataset, query) => {
-  // ==============================================================================================
-  // Variants
-  // ==============================================================================================
-
-  if (isVariantId(query)) {
-    const variantId = normalizeVariantId(query)
-    return Promise.resolve([
-      {
-        label: variantId,
-        value: `/variant/${variantId}?dataset=${dataset}`,
-      },
-    ])
-  }
+  if (dataset.startsWith('gnomad_sv')) {
+    // ==============================================================================================
+    // Structural Variants
+    // ==============================================================================================
 
-  if (isRsId(query)) {
-    const rsId = query
-    return Promise.resolve([
-      {
-        label: rsId,
-        value: `/variant/${rsId}?dataset=${dataset}`,
-      },
-    ])
-  }
+    if (isStructuralVariantId(query)) {
+      const structuralVariantId = query.toUpperCase()
+      return Promise.resolve([
+        {
+          label: structuralVariantId,
+          value: `/variant/${structuralVariantId}?dataset=${dataset}`,
+        },
+      ])
+    }
+  } else {
```

Comment:
This `else` should only include the checks if the query looks like a variant ID or rsID.

```js
if (dataset.startsWith('gnomad_sv')) {
  if (isStructuralVariantId(query)) { ... }
} else {
  if (isVariantId(query)) { ... }
  if (isRsid(query)) { ... }
}

if (isRegionId(query)) { ... }
...
```

We still want to be able to search by gene, region, etc. when an SV dataset is selected. Currently, if "gnomAD SVs v2.1" is selected, searching for "PCSK9" returns "Unable to get results" because this function returns undefined.

Same comment for the function in `graphql-api/src/queries/search.js`.